src/include/miopen/solver/conv_direct_naive_conv.hpp

```diff
@@ -36,29 +36,30 @@
 #include <vector>
 
 namespace miopen {
-
 namespace solver {
+namespace conv {
 
-bool ConvDirectNaiveConvIsAssemblyKernel(const ExecutionContext&, const ProblemDescription&);
-std::string ConvDirectNaiveConvKernelName(const ProblemDescription&);
+bool ConvDirectNaiveConvIsAssemblyKernel(const ExecutionContext&,
+                                         const miopen::conv::ProblemDescription&);
+std::string ConvDirectNaiveConvKernelName(const miopen::conv::ProblemDescription&);
 std::string ConvDirectNaiveConvKernelFile(const ExecutionContext& ctx,
-                                          const ProblemDescription& problem);
+                                          const miopen::conv::ProblemDescription& problem);
 std::string ConvDirectNaiveConvCompileOption(const ExecutionContext& ctx,
-                                             const ProblemDescription& problem);
+                                             const miopen::conv::ProblemDescription& problem);
 bool ConvDirectNaiveConvIsApplicableByKernelType(const ExecutionContext&,
-                                                 const ProblemDescription&);
-
-bool IsInputFp32(const ProblemDescription&);
-bool IsInputFp16(const ProblemDescription&);
-bool IsInputBfp16(const ProblemDescription&);
-bool IsInputInt8(const ProblemDescription&);
-bool IsAccFp64(const ProblemDescription&);
-bool IsAccInt32(const ProblemDescription&);
-bool IsOutputFp32(const ProblemDescription&);
-bool IsOutputFp16(const ProblemDescription&);
-bool IsOutputBfp16(const ProblemDescription&);
-bool IsOutputInt8(const ProblemDescription&);
-bool IsOutputInt32(const ProblemDescription&);
+                                                 const miopen::conv::ProblemDescription&);
+
+bool IsInputFp32(const miopen::conv::ProblemDescription&);
+bool IsInputFp16(const miopen::conv::ProblemDescription&);
+bool IsInputBfp16(const miopen::conv::ProblemDescription&);
+bool IsInputInt8(const miopen::conv::ProblemDescription&);
+bool IsAccFp64(const miopen::conv::ProblemDescription&);
+bool IsAccInt32(const miopen::conv::ProblemDescription&);
+bool IsOutputFp32(const miopen::conv::ProblemDescription&);
+bool IsOutputFp16(const miopen::conv::ProblemDescription&);
+bool IsOutputBfp16(const miopen::conv::ProblemDescription&);
+bool IsOutputInt8(const miopen::conv::ProblemDescription&);
+bool IsOutputInt32(const miopen::conv::ProblemDescription&);
 
 namespace conv_internal {
```

Comment:
might be a good idea to rename it to `internal` as it's already nested inside `conv` namespace.

Reply:
Let's leave it as it is for now